WalletWasabi/Hwi2/Models/HwiEnumerateEntry.cs

```diff
@@ -0,0 +1,48 @@
+using NBitcoin;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using WalletWasabi.Hwi2.Exceptions;
+
+namespace WalletWasabi.Hwi2.Models
+{
+	public class HwiEnumerateEntry
+	{
+		public HardwareWalletVendors? Type { get; }
```

Comment:
I would prefer to have an `HardwareWalletVendors.Unknown` than a nullable.

Reply:
#metoo